docs/core/porting/libraries.md

```diff
@@ -13,23 +13,23 @@ ms.assetid: a0fd860d-d6b6-4659-b325-8a6e6f5fa4a1
 
 # Porting to .NET Core - Libraries
 
-With the release of .NET Core 1.0, there is an opportunity to port existing library code so that it can run cross-platform.  This article discusses the .NET Standard Library, unavailable technologies, how to account for the smaller number of APIs available on .NET Core 1.0, how to use the tooling that ships with .NET Core SDK Preview 2, and recommended approaches to porting your code.
+With the release of .NET Core 1.0, there is an opportunity to port existing library code so that it can run cross-platform. This article discusses the .NET Standard Library, unavailable technologies, how to account for the smaller number of APIs available on .NET Core 1.0, how to use the tooling that ships with .NET Core SDK Preview 2, and recommended approaches to porting your code.
```

Comment:
this topic needed a more extensive revision. Just updating the version was not enough here but @cartermp already has a PR here: https://github.com/dotnet/docs/pull/1792. But since you've cleaned the topic a bit of extra spaces, I'm willing to leave this one here.